app/src/main/java/com/example/habitsmasher/ui/history/EditHabitEventDialog.java

```diff
@@ -92,11 +112,35 @@ public void onClick(View view) {
                         eventComment,
                         _editHabitEvent.getId());
                 _errorText.setText("");
-                _habitEventListFragment.updateListAfterEdit(editedHabitEvent,_index);
+                _habitEventListFragment.editHabitEvent(editedHabitEvent,_index, _selectedImage);
 
                 // Close dialog
                 getDialog().dismiss();
             }
         });
     }
+
+    private void fetchEventImageFromDB() {
+        FirebaseStorage storage = FirebaseStorage.getInstance();
+        StorageReference storageReference = storage.getReference();
+
+        StorageReference ref = storageReference.child("images/" + _userId + "/" + _parentHabit.getId() + "/" + _editHabitEvent.getId() + "/eventImage");
+
+        final long ONE_MEGABYTE = 1024 * 1024;
+
+        ref.getBytes(ONE_MEGABYTE).addOnSuccessListener(new OnSuccessListener<byte[]>() {
+            @Override
+            public void onSuccess(byte[] bytes) {
+                Bitmap bm = BitmapFactory.decodeByteArray(bytes, 0, bytes.length);
+                _habitImage = bm;
```

Comment:
If the variable `bm` isn't used anywhere else other than this assignment statement, I think we can just inline the variable and get rid of it.

`_habitImage = BitmapFactory.decodeByteArray(bytes, 0, bytes.length);`